protocol/x/clob/ante/clob.go

```diff
@@ -106,6 +124,17 @@ func (cd ClobDecorator) AnteHandle(
 				return next(ctx, tx, simulate)
 			}
 
+			// HOTFIX: Ignore any short-term clob messages in a transaction with a timeout height.
+			if timeoutHeight > 0 {
```

Comment:
For my understanding why can't we just return error here for ST placement in CheckTx? 

```
			if timeoutHeight > 0 && ctx.IsCheckTx(){
				// return error
			}
			```

Reply:
Returned an error instead.

Reply:
Original reason for not adding an error, I'm wary of adding errors in a rolling hotfix but this should be fine since it just rejects during checktx.